src/lava/lib/optimization/solvers/generic/qp/processes.py

```diff
@@ -304,7 +304,7 @@ class ProjectedGradientNeuronsPIPGeq(AbstractProcess):
     process with this solver. Use the standard Lava Dense process.
     Intialize the ProjectedGradientNeuronsPIPGeq process.
     Implements the abstract behaviour
-    qp_neuron_state -= alpha*(a_in_qc + grad_bias + a_in_cn)
+    qp_neuron_state -= alpha*(a_in + grad_bias)

```

Comment:
Could you elaborate why you used a_in_cn previously, but is not needed anymore? Will this change the behavior related to the model used for the NeurIPS paper?

Reply:
no it won't. I optimized pymodel to use only 1 inport and 1 outport like the ncmodel